sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -222,51 +310,35 @@ export const logger: import("@azure/logger").AzureLogger;
 
 // @public
 export interface OrganizationDetails {
-    // Warning: (ae-forgotten-export) The symbol "AdministratorDetails" needs to be exported by the entry point index.d.ts
     adminDetails?: AdministratorDetails[];
     id?: string;
 }
 
-// @public (undocumented)
-export interface ParsedKeyVaultEntityIdentifier {
-    name: string;
-    vaultUrl: string;
-    version?: string;
-}
-
 export { PipelineOptions }
 
 // @public
-export interface ProxyOptions {
-    // (undocumented)
-    proxySettings?: string;
+export interface SecretProperties {
+    contentType?: string;
 }
 
 // @public
-export interface RetryOptions {
```

Comment:
Are we not exposing RetryOptions at all anymore?

Reply:
We're not in keys nor secrets

Reply:
If user wants to customize the pipeline, they are to use the types from core-http if needed